AdornUI-Stylesheets/input.css

```diff
@@ -0,0 +1,49 @@
+.form-group {
+  margin: 2rem;
+}
+.form-control {
+  font-size: 2rem;
+  height: 3rem;
+  border-radius: 5px;
+  border: 0.5px solid var(--primary-bg-color);
+  padding: 1rem;
+  color: var(--primary-text-color);
+}
+.form-label {
+  font-size: 2rem;
+  color: var(--primary-text-color);
+  font-weight: 900;
+}
+.form-text {
+  font-size: 1.2rem;
+  font-weight: 600;
+}
+.text-mutated {
+  color: var(--primary-text-color);
+}
+.text-error {
+  color: var(--danger-color);
+}
+.form-field-required:after {
+  content: "*";
+  color: var(--danger-color);
+  font-size: 3rem;
+}
+#email-input:focus {
+  border: 2px solid var(--primary-text-color);
+}
+#password-input:focus {
+  border: 2px solid var(--primary-text-color);
+}
+::placeholder {
+  color: var(--grey);
+  opacity: 1;
+}
+#select-input {
```

Comment:
Using id here would mean we would be able to associate with only 1 element. What should the user do if they have 2 select inputs? Plus adding any id here also means any user of Adorn UI is restricted to use these particular ids only only.

Similarly for `#password-input`, `#email-input`.

Reply:
So then I have to remove the 'for' attribute as well right in the label? I added ids considering that I will need to mention it in for part of label. Is it ok not to have 'for' attribute? Because in 'for' we can only mention id right?